index.html

```diff
@@ -2133,10 +2258,38 @@ <h2>
         </dt>
         <dd>
           A boolean that indicates whether the <a>user agent</a> SHOULD collect
-          and return a shipping address as part of the payment request. For
-          example, this would be set to true when physical goods need to be
-          shipped by the merchant to the user. This would be set to false for
-          the purchase of digital goods.
+          and return a shipping address as part of the payment request (e.g.,
+          when physical goods need to be shipped by the payee to the user). The
+          <a>user agent</a> returns the shipping address in the
+          {{PaymentRequest}}'s {{PaymentRequest/shippingAddress}}
+          attribute and the {{PaymentResponse}}'s
+          {{PaymentResponse/shippingAddress}} attribute. The former is subject
+          to a <a>shipping redact list</a> per the <a>shipping address changed
+          algorithm</a>.
+          <aside class="note">
+            <p>
+              A payee may not need the full shipping address of a user until the
+              payment request is completed. For example, just having "country"
+              may be sufficient to calculate shipping cost. A payee sensitive to
+              the user's privacy can use <a>requestShippingAddressParts</a> to
+              restrict the initial set of address parts returned by the
+              <a>user agent</a>.
+            </p>
+          </aside>
+        </dd>
+        <dt>
+          <dfn>requestShippingAddressParts</dfn> member
+        </dt>
+        <dd>
+          A sequence of {{AddressParts}} that specifies which fine-grained
+          parts of the shipping address that the payee wishes to receive when
+          the <a>requestShipping</a> member is true. An empty list means no
+          restriction, i.e. the <a>user agent</a> will return the full shipping
+          address, subject to the
+          <a>shipping redact list</a>. This field has no effect if the
```

Comment:
We generally try to avoid the word "field", as it's ambiguous in the context of JS/WebIDL. Properties on dictionaries are formally referred to as "members".

```suggestion
          <a>shipping redact list</a>. The {{PaymentOptions/requestShippingAddressParts}} member has no effect if the
```